enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -124,11 +124,11 @@ const metabaseConfig = {
 ### JWT Authentication
```

Comment:
<img width="313" alt="image" src="https://github.com/user-attachments/assets/c1539dd3-3deb-4b3b-9f85-a32f1bea6877">
 
Press the magic button ^
It will show a hugely better diff